openedx/core/djangoapps/zendesk_proxy/utils.py

```diff
@@ -53,15 +68,20 @@ def _std_error_message(details, payload):
         log.error(_std_error_message("zendesk not configured", payload))
         return status.HTTP_503_SERVICE_UNAVAILABLE
 
+    if group:
+        if group in settings.ZENDESK_GROUP_ID_MAPPING:
+            group_id = settings.ZENDESK_GROUP_ID_MAPPING[group]
+            data['ticket']['group_id'] = group_id
+        else:
+            msg = u"Group ID not found for group {}. Please update ZENDESK_GROUP_ID_MAPPING".format(group)
+            log.error(_std_error_message(msg, payload))
+            return status.HTTP_503_SERVICE_UNAVAILABLE
```

Comment:
If the client sends something that doesn't match a configured group, shouldn't it be a 400?